pages/course.tsx

```diff
@@ -1,18 +1,41 @@
-import React from "react";
+import React, { useEffect } from "react";
 import styles from "../styles/courses.module.scss";
 import Head from "next/head";
 import Layout from "../components/layout/layout";
-import EmailCourse from "../components/emailForm/forms/emailCourse";
 import FormInput from "../components/emailForm/formInput";
 import FAQ from "../components/faq/faq";
-
 import questions from "../components/faq/course_faq";
 import { eventPropNewsletter, eventSignup } from "../core/constants";
+import InfoCard from "../components/cards/infoCard/infoCard";
+import ModuleCard from "../components/cards/moduleCard/moduleCard";
+import PricingCard from "../components/cards/pricingCard/pricingCard";
+import Quote from "../components/quote/quote";
+import info from "../components/cards/infoCard/info";
+import modules from "../components/cards/moduleCard/modules";
+import queryString from "query-string";
+import {
+  package_basic,
+  package_complete,
+  package_exclusive,
+} from "../components/cards/pricingCard/packages";
 
 const form_url =
   "https://gmail.us2.list-manage.com/subscribe/post?u=ff73d806dd2f49da87ede8337&amp;id=ed4e712aca&amp;SIGNUP=FlutterMovieCourse";
 
-const courses = () => {
+const course = () => {
+  const getAffcode = () => {
+    const qs = queryString.parse(window.location.search);
+    const affcode = qs.affcode;
+
+    if (affcode) {
+      window.sessionStorage.setItem("affcode", JSON.stringify(affcode));
```

Comment:
I think we should use localStorage instead as we want it to be stored even if the user closes the sites and decides to buy it the next day.

Maybe a maximum of 30 days if it's easy to implement